pages.fr/common/pip.md

```diff
@@ -3,30 +3,34 @@
 > Gestionnaire des paquets pour Python.
 > Plus d'informations : <https://pip.pypa.io>.
 
-- Installe un paquet :
+- Installe un paquet:
 
 `pip install {{paquet}}`
 
-- Installe une version particulière d'un paquet :
+- Installe une version particulière d'un paquet:
 
 `pip install {{paquet}}=={{version}}`
 
-- Met à jour un paquet :
+- Installer un package dans le répertoire de l'utilisateur au lieu de l'emplacement par défaut à l'échelle du système:
```

Comment:
@Nico385412 could you please have a look at this? I think it was translated by some online service.

Reply:
You're right, the sentence sounds weird in french.

I'd rather say 

```
Installe un paquet dans le répertoire utilisateur au lieu de l'emplacement par défaut système:
```

- Replace `Installer` by `Installe` for imperative 
- Replace `package` by his french translation `paquet`
- Replace `répertoire de l'utilisateur` by `répertoire utilisateur` their is no need to add `de l'` 
![image](https://user-images.githubusercontent.com/9024389/139678951-b211d81e-ce19-46eb-8f6c-5d3dc2322160.png)
- Replace `l'emplacement par défaut à l'échelle du système:` by `l'emplacement par défaut système:`
> the words ` à l'échelle du système` does not make sense here. In french you use `à l'echelle d(e/u) XXXX` for a relative comparison.
![image](https://user-images.githubusercontent.com/9024389/139679976-deeec450-4419-4969-9a26-5cc072dc6a80.png)

what du you think ? ^^"

Reply:
Thanks!